homeassistant/components/flunearyou/__init__.py

```diff
@@ -36,17 +41,22 @@ async def async_setup_entry(hass, entry):
     latitude = entry.data.get(CONF_LATITUDE, hass.config.latitude)
     longitude = entry.data.get(CONF_LONGITUDE, hass.config.longitude)
 
-    async def async_update(api_category):
+    async def async_update(api_category: str) -> dict[str, Any]:
         """Get updated date from the API based on category."""
         try:
             if api_category == CATEGORY_CDC_REPORT:
-                return await client.cdc_reports.status_by_coordinates(
+                data = await client.cdc_reports.status_by_coordinates(
+                    latitude, longitude
+                )
+            else:
+                data = await client.user_reports.status_by_coordinates(
                     latitude, longitude
                 )
-            return await client.user_reports.status_by_coordinates(latitude, longitude)
         except FluNearYouError as err:
             raise UpdateFailed(err) from err
 
+        return cast(Dict[str, Any], data)
```

Comment:
Since you are the owner of the library that is being implemented here, could we update the typing upstream instead?

Reply:
Even with [an updated `pyflunearyou`](https://github.com/bachya/pyflunearyou/releases/tag/2.0.1), not including a specific typecast results in:

```
homeassistant/components/flunearyou/__init__.py:58: error: Returning Any from function declared to return "Dict[str, Any]"
```

Reply:
The PEP561 `py.typed` file missing in the upstream package?